org.opent2t.sample.multisensor.superpopular/test/multisensorTests.js

```diff
@@ -284,6 +284,17 @@ function runMultisensorTests(settings) {
             });
     });
 
+        test.serial('GetBrightnessChange', t => {
```

Comment:
Doh! thanks missed seeing it up there. 

---
In reply to: [106059745](https://github.com/openT2T/translators/pull/210#discussion_r106059745) [](ancestors = 106059745)